RetroBar/PropertiesWindow.xaml.cs

```diff
@@ -363,6 +365,20 @@ private void CbAutoStart_OnChecked(object sender, RoutedEventArgs e)
             }
         }
 
+        private void CbOverrideHotkeys_OnChecked(object sender, RoutedEventArgs e)
+        {
+            var chkBox = (System.Windows.Controls.CheckBox)sender;
+
+            if (chkBox.IsChecked.Equals(false))
+            {
+                _explorerMonitor.explorerMonitorWindow.UnregisterHotkeys();
+            }
+            else
+            {
+                _explorerMonitor.explorerMonitorWindow.RegisterHotkeys();
+            }
+        }
```

Comment:
Instead of doing this (and needing to plumb ExplorerMonitor into here, and needing to make explorerMonitorWindow public), instead ExplorerMonitor should add a PropertyChanged listener for the Settings instance, and register/unregister based on the setting change.